pkg/cmd/lint-epic-issue-refs/lint_epic_issue_refs.go

```diff
@@ -0,0 +1,319 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"reflect"
+	"regexp"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+var (
+	fixIssueRefRE        = regexp.MustCompile(`(?im)(?i:close[sd]?|fix(?:e[sd])?|resolve[sd]?):?\s+(?:(?:(#\d+)|([\w.-]+[/][\w.-]+#\d+)|([A-Z]+-\d+))[,.;]?(?:[ \t\n\r]+|$))+`)
+	informIssueRefRE     = regexp.MustCompile(`(?im)(?:see also|informs):?\s+(?:(?:(#\d+)|([\w.-]+[/][\w.-]+#\d+)|([A-Z]+-\d+))[,.;]?(?:[ \t]+|$))+`)
+	epicRefRE            = regexp.MustCompile(`(?im)epic:?\s+(?:([A-Z]+-[0-9]+)[,.;]?(?:[ \t]+|$))+`)
+	epicNoneRE           = regexp.MustCompile(`(?im)epic:?\s+(?:(none)[,.;]?(?:[ \t]+|$))+`)
```

Comment:
Good find on adding the `\n\r`. Let's fix it for the other three regexes too. Also, I think the `+` after the `[ \t\n\r]` can be removed. The regexes only need to verify that there is at least one separating character there.

```suggestion
	fixIssueRefRE        = regexp.MustCompile(`(?im)(?i:close[sd]?|fix(?:e[sd])?|resolve[sd]?):?\s+(?:(?:(#\d+)|([\w.-]+[/][\w.-]+#\d+)|([A-Z]+-\d+))[,.;]?(?:[ \t\n\r]|$))+`)
	informIssueRefRE     = regexp.MustCompile(`(?im)(?:see also|informs):?\s+(?:(?:(#\d+)|([\w.-]+[/][\w.-]+#\d+)|([A-Z]+-\d+))[,.;]?(?:[ \t\n\r]|$))+`)
	epicRefRE            = regexp.MustCompile(`(?im)epic:?\s+(?:([A-Z]+-[0-9]+)[,.;]?(?:[ \t\n\r]|$))+`)
	epicNoneRE           = regexp.MustCompile(`(?im)epic:?\s+(?:(none)[,.;]?(?:[ \t\n\r]|$))+`)
```

Reply:
I added `\n\r` to each of the other RegExes, but I kept the `+`, because tests were failing without it.